arch/Kconfig

```diff
@@ -403,6 +405,24 @@ config DYNAMIC_INTERRUPTS
 	  interrupt-related data structures to RAM instead of ROM, and
 	  on some architectures increase code size.
 
+config SHARED_INTERRUPTS
+	bool "Set this to enable support for shared interrupts"
+	default n
+	depends on GEN_SW_ISR_TABLE
+	depends on ARCH_HAS_SHARED_IRQ_SUPPORT
+	select EXPERIMENTAL
+	help
```

Comment:
here it is important to stress that adding support for shared IRQs is going to non-negligibly increase the image size and maybe give a ballpark estimation of that.

Reply:
Should be a bit better. If it still doesn't stress enough the idea that enabling `SHARED_INTERRUPTS` will non-negligibly increase the image size then I'll also try to add an estimation of the increase in size like you initially suggested.